### PR TITLE
Fix cmake build of MK2.5 and MK25S

### PR DESCRIPTION
**Pull request by 3d-gussner**
Got broken with https://github.com/prusa3d/Prusa-Firmware/commit/0b23ab8e092767771adb2761d6ea1384c462dfd3#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL476

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -565,7 +565,7 @@ function(fw_add_variant variant_name)
       # Patched binary with pre-baked secondary language
       add_custom_command(
         OUTPUT ${FW_LANG_FINAL}.bin
-        COMMAND ${CMAKE_OBJCOPY} -O binary ${FW_LANG_BASE} ${FW_LANG_FINAL}.bin
+        COMMAND ${CMAKE_COMMAND} -E copy ${FW_LANG_PATCH}.bin ${FW_LANG_FINAL}.bin
         COMMAND "${Python3_EXECUTABLE}" ${CMAKE_SOURCE_DIR}/lang/lang-patchsec.py ${FW_LANG_BASE} ${LANG_BIN}
                 ${FW_LANG_FINAL}.bin
         DEPENDS ${FW_LANG_BASE} ${LANG_BIN}
```
Comment (vintagepc) [inline CMakeLists.txt]: should add `${FW_LANG_PATCH}.bin` to DEPENDS so the dependency tree is ordered correctly.

**Comment by vintagepc** [comment]
Confirmed the same change fixes it locally for me as well. Just one change required.

File: CMakeLists.txt (hunk 1 above)
Comment (3d-gussner) [reply CMakeLists.txt]: Done

**Comment by 3d-gussner** [comment]
@vintagepc Thanks for the review.

**Review by vintagepc — APPROVED** [review]

**Pull request merged**